server/channels/app/support_packet.go

```diff
@@ -52,28 +42,40 @@ func (a *App) GenerateSupportPacket(c request.CTX, options *model.SupportPacketO
 	for name, fn := range functions {
 		fileData, err := fn(c)
 		if err != nil {
-			c.Logger().Error("Failed to generate file for support package", mlog.Err(err), mlog.String("file", name))
-			warnings = append(warnings, err.Error())
+			c.Logger().Error("Failed to generate file for Support Packet", mlog.String("file", name), mlog.Err(err))
+			warnings = multierror.Append(warnings, err)
 		}
 
 		if fileData != nil {
 			fileDatas = append(fileDatas, *fileData)
 		}
 	}
 
+	if cluster := a.Cluster(); cluster != nil && *a.Config().ClusterSettings.Enable {
+		files, err := cluster.GenerateSupportPacket(c, options)
```

Comment:
Due to the CPU profile generation for the cluster nodes, generating the Support Packet takes an additional 5 seconds, increasing the total duration to at least 10 seconds. I wonder if we should introduce a separate go routine.

Reply:
Moved into a separate go routine via 25339bcedc